pkg/runtime/ecosystem/javascript.go

```diff
@@ -34,6 +36,13 @@ func (e *JavaScript) Add(artifact *buildplan.Artifact, artifactSrcPath string) (
 	}
 	var packageName string
 	for _, file := range files {
+		if file.Name() == "runtime.json" {
+			err = injectEnvVar(file.AbsolutePath(), "NPM_CONFIG_PREFIX", "${INSTALLDIR}/usr")
+			if err != nil {
+				return nil, errs.Wrap(err, "Unable to add NPM_CONFIG_PREFIX to runtime.json")
+			}
+			continue
+		}
```

Comment:
Why is this needed? The intend is that us running `npm` frees us up from this responsibility.